.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Create a bug report to help us improve
+title: "Bug Summary"
+labels: "bug"
+assignees: ""
+---
+
+**Describe the bug**
+
+<!-- A clear and concise description of what the bug is. -->
+
+**To Reproduce**
+
+Steps to reproduce the behavior:
+
+1. ...
+2. ...
+3. ...
+
+**Expected behavior**
+
+<!-- A clear and concise description of what you expected to happen. -->
+
+**System [please complete the following information]:**
+
+-   OS: e.g. [Ubuntu 18.04]
+-   Language Version: [e.g. Python 3.8]
+-   Virtual environment: [e.g. Conda]
```

Comment:
Maybe nit-picky, but using Conda as an example here seems a bit strange. Maybe change to `Poetry version: [e.g. 1.1.13]`

Reply:
no worries, i ll fix that as well :)